dev-tools/create_bwc_index.py

```diff
@@ -133,16 +133,19 @@ def start_node(version, release_dir, data_dir, repo_dir, tcp_port=DEFAULT_TRANSP
   logging.info('Starting node from %s on port %s/%s, data_dir %s' % (release_dir, tcp_port, http_port, data_dir))
   if cluster_name is None:
     cluster_name = 'bwc_index_' + version
-    
+  if version.startswith('5'):
```

Comment:
perhaps this check should be for versions before 5 instead, otherwise we'll need to fix this for 6.0 as well